.github/workflows/node-ci.yml

```diff
@@ -214,8 +214,24 @@ jobs:
                 os: [ubuntu-latest]
                 node-version: [20.x]
         steps:
+            # First, we need to see if any releases are in progress.
+            # We do not want to try and publish anything if a publish is
+            # pending. We fail here, but we make sure to update the
+            # PR comment later.
+            - name: Check for release
+              id: check-release
+              run: |
+                  release_count=$(gh run list --workflow release.yml --json status --jq '[.[] | select(.status == "in_progress")] | length')
+                  echo "release_count=$release_count" >> $GITHUB_OUTPUT
+                  if [ "$release_count" -ne 0 ]; then
+                    echo "Error: There are $release_count releases in progress."
+                    exit 1
+                  fi
+
             # We need to checkout all history, so that the changeseat tool can diff it
             - name: Checkout current commit
+              # Only bother if there are no releases in progress
+              if: steps.check-release.outputs.release-count == '0'
```

Comment:
**note:** The job failure should mean this doesn't run, but no harm in being explicit.